apps/client/src/components/Layout/Sider/SiderCategory/index.module.css

```diff
@@ -26,15 +29,15 @@
 
 .linkcontent > :first-child {
   margin-right: 8px;
+  font-size: 1.25em;
 }
 
 .link > span {
   margin-right: 8px;
 }
 
 .link:hover {
-  background: grey;
-  text-decoration: underline;
+  background: rgba(255, 255, 255, 0.2);
```

Comment:
You should variabilise this in index.css, so that we can provide light and dark variants. Named something like link-hover-background